pkg/ansible/paramconv/paramconv.go

```diff
@@ -83,10 +83,10 @@ func ToSnake(s string) string {
 	}
 	bits := []string{}
 	n := ""
-	real_i := -1
+	real := -1
```

Comment:
Nit: The implication of the name `real_i` is that it is an index and is used in contrast with the variable `i`.

Maybe rename this to `iReal` to keep the tie-in with `i`?